mage_ai/tests/data_cleaner/cleaning_rules/test_remove_outliers.py

```diff
@@ -1,6 +1,8 @@
+import statistics
```

Comment:
What's this library used for?

Reply:
Removed, it was auto imported unknowingly